docs/getting-started/first-release.md

```diff
@@ -0,0 +1,199 @@
+
+Once the application components are ready, the Iter8 controller automatically configures the routing by creating an Istio `VirtualService`. It is configured to route all traffic to the only deployed version, `httpbin-0`.
+
+### Verify routing
+
+You can verify the routing configuration by inspecting the `VirtualService`:
+
+```shell
+kubectl get virtualservice httpbin -o yaml
+```
+
+You can also send requests from a pod within the cluster:
+
+1. Create a `sleep` pod in the cluster from which requests can be made:
+```shell
+curl -s https://raw.githubusercontent.com/iter8-tools/docs/v0.17.3/samples/kserve-serving/sleep.sh | sh -
+```
+
+2. Exec into the sleep pod:
+```shell
+kubectl exec --stdin --tty "$(kubectl get pod --sort-by={metadata.creationTimestamp} -l app=sleep -o jsonpath={.items..metadata.name} | rev | cut -d' ' -f 1 | rev)" -c sleep -- /bin/sh
+```
+
+3. Send requests:
+```shell
+curl httpbin.default -s -D - | grep -e '^HTTP' -e app-version
+```
+
+The output includes the success of the request (the HTTP return code) and the version of the application that responded (the `app-version` response header). For example:
+
+```
+HTTP/1.1 200 OK
+app-version: httpbin-0
+```
+
+??? note "To send requests from outside the cluster"
+    To configure the release for traffic from outside the cluster, a suitable Iter8 `Gateway` is required. For example, this [sample gateway](https://raw.githubusercontent.com/kalantar/docs/release/samples/iter8-sample-gateway.yaml). When using the Iter8 `release` chart, set the `gateway` field to the name of your `Gateway`. Finally, to send traffic:
+
+    (a) In a separate terminal, port-forward the ingress gateway:
+    ```shell
+    kubectl -n istio-system port-forward svc/istio-ingressgateway 8080:80
+    ```
+    (b) Send requests using the `Host` header:
+    ```shell
+    curl -H 'Host: httpbin.default' localhost:8080 -s -D - | grep -e '^HTTP' -e app-version
+    ```
+
+## Deploy candidate
+
+A candidate can deployed by simply adding a second version to the list of versions comprising the application:
+
+```shell
+cat <<EOF | helm upgrade --install httpbin $CHARTS/release -f -
+environment: deployment-istio
+application: 
+  versions:
+  - metadata:
+      labels:
+        app.kubernetes.io/version: v0
+    image: kennethreitz/httpbin
+  - metadata:
+      labels:
+        app.kubernetes.io/version: v1
+    image: kennethreitz/httpbin
+  strategy: blue-green
+EOF
+```
+
+??? note "About the candidate version"
+    In this tutorial, the candidate image is the same as the one for the primary version. In a real world example, it would be different. The version label (`app.kubernetes.io/version`) can be used to distinguish between versions.
```

Comment:
```suggestion
    In this tutorial, the candidate image is the same as the one for the primary version. In a real world example, it would be different. In this case, the Kubernetes version label (`app.kubernetes.io/version`) is be used to distinguish between versions but other kinds of metadata can be used as well.
```